package.json

```diff
@@ -26,24 +26,21 @@
   ],
   "peerDependencies": {
     "react-native": "~0.42.0-rc.2",
-    "react": "~15.4.0-rc.4"
+    "react": "~15.4.1"
   },
   "devDependencies": {
     "https-proxy-agent": "^1.0.0",
     "jasmine": "^2.5.2",
     "jasmine-reporters": "^2.2.0",
-    "react": "~15.4.0-rc.4",
+    "react": "~15.4.1",
     "react-native": "~0.42.0-rc.2",
     "selenium-webdriver": "2.48.x"
   },
   "dependencies": {
     "chalk": "^1.1.3",
     "glob": "^7.0.3",
     "shelljs": "^0.7.0",
-    "uuid": "^2.0.1",
-    "xml-parser": "^1.2.1",
-    "yeoman-environment": "^1.5.3",
```

Comment:
I think we're still using yeoman for project generators.  I filed https://github.com/ReactWindows/react-native-windows/issues/979 to create an alternative generator mechanism.